cmd/root.go

```diff
@@ -42,34 +41,29 @@ func CreateConfig() {
 	flag.Parse()
 
 	if cfgFile != "" {
-		log.Printf("reading configs from file: %v", cfgFile)
+		log.Printf("Reading configs from file: %v", cfgFile)
 		file, err := os.Open(cfgFile)
 		if err != nil {
-			log.Print("can't open config file: ", err)
+			log.Print("Can't open config file: ", err)
 		}
 		defer file.Close()
 		decoder := json.NewDecoder(file)
 		err = decoder.Decode(&opts)
 		if err != nil {
-			log.Print("can't decode config JSON: ", err)
+			log.Print("Can't decode config JSON: ", err)
 		}
 	}
-	flag.Parse()
 }
 
 func RunCmdRoot() {
-
-	stop := make(chan struct{})
-	done := make(chan struct{})
+	//stop := make(chan struct{})
```

Comment:
Can this be deleted?

Reply:
yes. i wanted to create a draft pull request but ooops.
there's still a bit of cleanuo to do, and i only ran a few manual tests. i want to test this properly with a real app in a kube environment.